app/api/helpers/permission_manager.py

```diff
@@ -354,8 +354,8 @@ def permission_manager(view, view_args, view_kwargs, *args, **kwargs):
         try:
             order = Order.query.filter_by(identifier=view_kwargs['order_identifier']).one()
         except NoResultFound:
-            return NotFoundError({'parameter': 'order_identifier'}, 'Event not found').respond()
-        view_kwargs['id'] = order.identifier
+            return NotFoundError({'parameter': 'order_identifier'}, 'Order not found').respond()
+        view_kwargs['id'] = order.id
```

Comment:
For me, using `id` instead of `identifier` here threw an error, because it then compared order `id` with `identifier` later, in the library code.

Reply:
For me it is working. Not sure why it wasn't working for you.

Reply:
I tried using `id` instead of `identifier` and it's working now. I don't recall why it was not working then. Maybe because I was using a modified order schema.